src/lib/containers/widgets/facet-nav/FacetNav.tsx

```diff
@@ -192,9 +190,8 @@ const FacetNav: React.FunctionComponent<FacetNavProps> = ({
   } else if (isLoadingNewData) {
     return (
       <div className="SRC-loadingContainer SRC-centerContentColumn">
-        {loadingScreen}
         {asyncJobStatus?.progressMessage && (
-          <div>{asyncJobStatus.progressMessage} </div>
+          <div> <span className="spinner" /> {asyncJobStatus.progressMessage} </div>
```

Comment:
Is this a different implementation for the loading state because it has a custom message? Thanks.

Reply:
This could be a loadingScreen too (with the async message), I just had a spinner since the Table will always have the loading bar (so I thought 2 of these bars right next to each other might look strange, if this loader is ever shown!).  I'd be happy to change if you think a bar would be preferred.

Reply:
That's okay. I think it makes sense to have a spinner. Thanks.